pkg/apis/testharness/v1beta1/collector_test.go

```diff
@@ -3,6 +3,8 @@ package v1beta1
 import (
 	"strings"
 	"testing"
+
+	"gotest.tools/assert"
```

Comment:
please switch to testify (which will remove the new dependency as well with a go mod verify)

Reply:
Awesome, works even better!